code/modules/clothing/suits/hood.dm

```diff
@@ -77,6 +60,7 @@
 				to_chat(H,"<span class='warning'>You're already wearing something on your head!</span>")
 				return
 			else if(H.equip_to_slot_if_possible(hood,slot_head,0,0,1))
+				hood.forceMove(H)
```

Comment:
Uhhhh, forcemoving the hood into the mob?

Reply:
`equip_to_slot_if_possible` should called `equip_to_slot`, which already sets the item's `loc`.

Reply:
Because it doesn't actually change the location of the hood to the humanoid. The loc stays the same. Aka inside of the hoodie.

Reply:
I'm an idiot. You'll see why in the commit